db/migrate/20241216140931_add_tax_status_to_invoices.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTaxStatusToInvoices < ActiveRecord::Migration[7.1]
+  def change
+    add_column :invoices, :tax_status, :integer, null: false, default: 1
```

Comment:
What do you think of using an enum type here? (using `create_enum`)

Reply:
Updated